### PR TITLE
Chore: fixed typos and aligned instructions in EKS, GKE, and AKS readmes

### PR DESCRIPTION
**Pull request by MaggieXJZhang**
This PR fixes lots of the typos and inconsistencies between the three CSPs:

* A few typo fixes + rewording from https://github.com/NVIDIA/nvidia-terraform-modules/issues/6
* Added a note about stopping using changelog with pointer to the release page.
* Aligned all readmes to use the same from of `terraform plan -out tfplan` and added `tfplan` to gitignore.
        * Note: previously on AKS: `terraform plan -out plan.tf` and the `.tf` actually confuses terraform and causes an error.

### CODE REVIEW

File: CHANGELOG.md
Comment (evberrypi) [inline CHANGELOG.md:1]: Suggest to just delete the changelog. We have kept the release notes in the release page as well

File: aks/README.md [hunk 1]
```diff
@@ -51,8 +51,8 @@ We strongly encourage you [configure remote state](https://developer.hashicorp.c
     - Run `terraform init` to get started
 
 4. Run the module
-    - Run `terraform plan -out plan.tf` to view the proposed changes
-    - Run `terraform apply plan.tf` to apply the configuration. 
+    - Run `terraform plan -out tfplan` to view the proposed changes
+    - Run `terraform apply tfplan` to apply the configuration.
```
Comment (evberrypi) [inline aks/README.md:55]: THANK YOU!

File: CHANGELOG.md
Comment (MaggieXJZhang) [reply CHANGELOG.md:1]: deleted

File: aks/README.md (hunk 1 above)
Comment (MaggieXJZhang) [reply aks/README.md:55]: you're welcome

**Review by evberrypi — APPROVED** [review]

**Pull request merged**